client/app/configuration/configuration.controller.js

```diff
@@ -1,51 +1,72 @@
-angular.module('App').controller(
-  'ConfigurationCtrl',
-  class ConfigurationCtrl {
-    constructor(
-      $scope,
-      $route,
-      WucProducts,
-      constants,
-      User,
-    ) {
-      this.$scope = $scope;
-      this.$route = $route;
-      this.WucProducts = WucProducts;
-      this.constants = constants;
-      this.User = User;
-    }
-
-    $onInit() {
-      this.guides = this.constants.TOP_GUIDES;
-
-      this.User.getUser()
-        .then((user) => {
-          this.allGuides = _.get(
-            this.constants,
-            `urls.${user.ovhSubsidiary}.guides.all`,
-            this.constants.urls.FR.guides.all,
+angular
+  .module('App')
+  .controller(
+    'configurationCtrl',
+    class ConfigurationCtrl {
+      constructor(
+        $route,
+        $scope,
+
+        constants,
+        User,
+        WucProducts,
+      ) {
+        this.$route = $route;
+        this.$scope = $scope;
+
+        this.constants = constants;
+        this.User = User;
+        this.WucProducts = WucProducts;
+      }
+
+      $onInit() {
+        this.guides = this.constants.TOP_GUIDES;
+
+        this.currentGuides = [];
+        this.currentTypeOfGuide = null;
+
+        this.selectTypeOfGuide('domainHosting');
+
+        this.helpCenterURLs = Object.keys(this.constants.urls)
```

Comment:
Won't a `mapValues` would do ? 
```suggestion
        this.helpCenterURLs = _.mapValues(this.constants.url, (value, subsidiary) => _.get(value, 'support'))
```

Reply:
No it wouldn't:
- some keys are to be ignored (they are not subsidiaries 😅)
- my object is like this:
```js
{
  FR: 'ovh.fr/support',
  DE: 'ovh.de/support,
  ...
}
```

Reply:
Done !